sunpy/net/vso/vso.py

```diff
@@ -21,7 +21,6 @@
 import socket
 import itertools
 
-from datetime import datetime, timedelta
```

Comment:
Where all these statements just imported and never used?

Reply:
Because I did not remove them earlier.